From 8382e6bb0c76a8029493eae3f2d7a3dbfd0cfc12 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Mon, 5 Mar 2018 10:28:15 -0500
Subject: gx: Fix RANDR initialization for xserver 1.20

xf86DisableRandR() doesn't exist anymore, and we don't need it anyway,
the core code will notice that we set up RANDR ourselves.

Signed-off-by: Adam Jackson <ajax@redhat.com>

Downloaded from upstream commit
https://cgit.freedesktop.org/xorg/driver/xf86-video-geode/commit/?id=8382e6bb0c76a8029493eae3f2d7a3dbfd0cfc12

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 src/gx_driver.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gx_driver.c b/src/gx_driver.c
index ab57df1..4de336e 100644
--- a/src/gx_driver.c
+++ b/src/gx_driver.c
@@ -1426,7 +1426,9 @@ GXScreenInit(SCREEN_INIT_ARGS_DECL)
 
     /* Set up RandR */
 
+#if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) < 24
     xf86DisableRandR();         /* We provide our own RandR goodness */
+#endif
 
     /* Try to set up the shadow FB for rotation */
 
-- 
cgit v1.1

